tests/datavolume_test.go

```diff
@@ -811,6 +811,108 @@ var _ = Describe("[vendor:cnv-qe@redhat.com][level:component]DataVolume tests",
 		})
 	})
 
+	Describe("[rfe_id:1115][crit:high][vendor:cnv-qe@redhat.com][level:component][test] CDI Import from HTTP/S3", func() {
+
+		var (
+			dataVolume              *cdiv1.DataVolume
+			err                     error
+			originalImageName       = "cirros-qcow2.img"
+			testImageName           = "cirros-qcow2-1990.img"
+			tinyCoreIsoRateLimitURL = fmt.Sprintf("http://cdi-file-host.%s:82/cirros-qcow2-1990.img", f.CdiInstallNs)
+		)
+
+		BeforeEach(func() {
+			By("Prepare the file")
+			fileHostPod, err := utils.FindPodByPrefix(f.K8sClient, f.CdiInstallNs, utils.FileHostName, "name="+utils.FileHostName)
+			_, _, err = f.ExecCommandInContainerWithFullOutput(fileHostPod.Namespace, fileHostPod.Name, "http",
+				"/bin/sh",
+				"-c",
+				"cp /tmp/shared/images/"+originalImageName+" /tmp/shared/images/"+testImageName)
+			Expect(err).To(BeNil())
+		})
+
+		AfterEach(func() {
+			By("Delete DV")
+			err = utils.DeleteDataVolume(f.CdiClient, f.Namespace.Name, dataVolume.Name)
+			Expect(err).ToNot(HaveOccurred())
+
+			By("Cleanup the file")
+			fileHostPod, err := utils.FindPodByPrefix(f.K8sClient, f.CdiInstallNs, utils.FileHostName, "name="+utils.FileHostName)
+			_, _, err = f.ExecCommandInContainerWithFullOutput(fileHostPod.Namespace, fileHostPod.Name, "http",
+				"/bin/sh",
+				"-c",
+				"rm -f /tmp/shared/images/"+testImageName)
+			Expect(err).To(BeNil())
+
+			Eventually(func() bool {
```

Comment:
This is fine, and works great, but Tomasz has a PR that adds a WaitForPVCDeleted function to the utils that does exactly this. Depending on which one gets merged first, you might want to use that function.

Reply:
right, I'll check it

Reply:
the util is in PR #1257

Reply:
https://github.com/kubevirt/containerized-data-importer/blob/master/tests/utils/pvc.go#L224 Tomasz's PR has to deal with the CDI controller creating a new PVC and has to verify the new one is different from the original one. But I think you can just use the util.WaitPVCDeleted which is essentially the same as this.

Reply:
right - updated